src/components/shared/ReconfigurationStrategyOption.tsx

```diff
@@ -0,0 +1,66 @@
+import { CheckCircleFilled } from '@ant-design/icons'
+import { ThemeContext } from 'contexts/themeContext'
+import { CSSProperties, useContext } from 'react'
+
+import { Strategy } from 'constants/ballotStrategies/ballotStrategies'
+
+// This component is either the selected option which opens the drawer of all options, or is an option within that drawer.
+export default function ReconfigurationStrategyOption({
```

Comment:
Awesome component!

Reply:
**question** will this ever be used outside of the Drawer component?

Wondering whether the drawer deserves its own directory, and this file be moved there... I think this Option component is nicely abstracted though and can exist standalone, but still might be worth logically grouping these components to help make the codebase easier to understand.

Reply:
Yep, the card is used outside the drawer in the v2 flow

Reply:
beautiful ✨